pkg/codegen/internal/test/sdk_driver.go

```diff
@@ -53,11 +55,11 @@ var sdkTests = []sdkTest{
 		Directory:   "simple-resource-schema-custom-pypackage-name",
 		Description: "Simple schema with local resource properties and custom Python package name",
 	},
-	// TODO[pulumi/pulumi#7072]: Enable test after codegen support for all languages lands.
-	// {
-	// 	Directory:   "simple-methods-schema",
-	// 	Description: "Simple schema with methods",
-	// },
+	{
+		Directory:   "simple-methods-schema",
+		Description: "Simple schema with methods",
+		Skip:        codegen.NewStringSet("dotnet"),
```

Comment:
I think you'll need to skip `"docs"` as well, or add the test data files for them.

Reply:
ah, good catch!